### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by darrenodi**
In this branch, I did the following 

-Added the contact form which submits to Form Spree
-Completed the mobile version of the Portfolio project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -232,5 +232,44 @@ <h2 class="primary-text">About me</h2>
...
+        <li>
+          <input type="text" name="name" required="required" placeholder="Full name" maxlength="30">
+        </li>
+        <li> 
+          <input type="email" name="email" required="required" placeholder="Email address">
+        </li>
+        <li>
+          <textarea name="textarea" id="textarea" rows="10" cols="30"  
+          required="required" placeholder="Enter text here" maxlength="500">Enter text here</textarea>
+        </li>
```
Comment (dasileker) [inline index.html:251]: - [ ] The required argument here is not working, please fix it

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @darrenodi**, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- You can just do **`required`** at the end of the tag instead of **`required="required"`**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (darrenodi) [reply index.html]: Done. It wasn't working because I added value to it. I've taken it out

**Review by dasileker — APPROVED** [review]
### **Hi @darrenodi**, 

### **Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it  ⛴
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**